accessibility-swarms.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: Accessibility Swarms
+---
+Accessibility Swarms are a practice that we carry out in BBC News.
+
+Swarms take place on **all new front-end code**, this means any changes to HTML, CSS or JavaScript code.
+
+We do this because **accessibility is a team responsibility**.
+
+## When should you carry out a swarm?
+- Before a ticket moves from Development to Test
+- When HTML, CSS & JS is code complete
+- Some developer testing with some assistive technology should already have taken place
+
+## How many people do you need?
+### If it’s a new component or feature
+Carry out a **team swarm**, with 4 to 6 people.
```

Comment:
`with 4 to 6 people` can be restrictive, for small teams. It would be good to explain instead which disciplines to be included, e.g. a Tester, BA, UX Designer, Developer. That the majority of the team should be involved.

Reply:
Will add a note about the number of people. Disciplines are covered in 'Who can take part'

Reply:
In fact I'll just combine the section with 'Who can take part' 😄